<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>804</width>
    <height>527</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#fff;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="vodZnach">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>261</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Введите значения переменных a, b, c, d:</string>
    </property>
   </widget>
   <widget class="QLabel" name="pic1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>251</width>
      <height>161</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../../Downloads/photo1710771600.jpeg</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="doGraph">
    <property name="geometry">
     <rect>
      <x>159</x>
      <y>245</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Построить график</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>215</y>
      <width>51</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="labelA">
       <property name="text">
        <string>a=</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelB">
       <property name="text">
        <string>b=</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelC">
       <property name="text">
        <string>c=</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelD">
       <property name="text">
        <string>d=</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="DmensheO">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>410</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="varA">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>216</y>
      <width>108</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="varB">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>244</y>
      <width>108</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="varC">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>272</y>
      <width>108</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="varD">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>300</y>
      <width>108</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="pic2">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>30</y>
      <width>441</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCustomPlot" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>130</y>
      <width>441</width>
      <height>311</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QLabel" name="reshenie">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>0</y>
      <width>371</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>804</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
